api_app/tests_ma/test_api/test_routes/test_workspaces.py

```diff
@@ -820,3 +832,74 @@ async def test_delete_user_resource_returns_resource_id(self, _, __, ___, get_us
 
         assert response.status_code == status.HTTP_200_OK
         assert response.json()["resourceId"] == user_resource.id
+
+    # [PATCH] /workspaces/{workspace_id}/workspace-services/{service_id}/user-resources/{resource_id}
+    @ patch("api.dependencies.workspaces.UserResourceRepository.get_user_resource_by_id", side_effect=EntityDoesNotExist)
+    async def test_patch_user_resources_returns_404_if_user_resource_does_not_exist(self, _, app, client):
+        response = await client.patch(app.url_path_for(strings.API_UPDATE_USER_RESOURCE, workspace_id=WORKSPACE_ID, service_id=SERVICE_ID, resource_id=USER_RESOURCE_ID), json='{"enabled": true}')
+        assert response.status_code == status.HTTP_404_NOT_FOUND
+
+    # [PATCH] /workspaces/{workspace_id}/workspace-services/{service_id}/user-resources/{resource_id}
+    @ patch("api.dependencies.workspaces.UserResourceRepository.get_user_resource_by_id", return_value=sample_user_resource_object())
+    @ patch("api.dependencies.workspaces.WorkspaceServiceRepository.get_workspace_service_by_id", side_effect=EntityDoesNotExist)
+    @ patch("api.dependencies.workspaces.WorkspaceRepository.get_workspace_by_workspace_id", return_value=sample_workspace())
+    async def test_patch_user_resources_returns_404_if_ws_service_does_not_exist(self, _, __, ___, app, client):
```

Comment:
This test won't be needed once we remove the workspace service parameter from the UR patch